.circleci/config.yml

```diff
@@ -106,8 +138,40 @@ jobs:
       - store_artifacts:
           path: integration_tests/screenshots
 
+  scenario_test:
+    executor: scenario-testing
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/app
+      - run:
+          name: Update npm
+          command: sudo npm install -g npm@latest
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package-lock.json" }}
+      - run:
+          name: Install Dependencies
+          command: npm ci --no-audit
+      - run:
+          command: npm run build
+      - run:
+          name: Run the node app
+          command: npm run start-scenarios
+          background: true
+      - run:
+          name: Wait for node app to start
+          command: sleep 5
+      - run:
+          name: Run scenario tests
+          command: npm run int-test-scenarios
+      - store_test_results:
+          path: test_results
+      - store_artifacts:
+          path: integration_tests/videos
+      - store_artifacts:
+          path: integration_tests/screenshots
+
```

Comment:
Can we run the built image rather than installing dependencies etc again? i.e. run the scenario tests after the `build_docker` job?